docs/download.md

```diff
@@ -43,7 +43,7 @@ for your hardware's minimum driver version.
 ### Download v21.12.0
 * Download the [RAPIDS
   Accelerator for Apache Spark 21.12.0 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/21.12.0/rapids-4-spark_2.12-21.12.0.jar)
-* Download the [RAPIDS cuDF 21.12.0 jar](https://repo1.maven.org/maven2/ai/rapids/cudf/21.12.0/cudf-21.12.0-cuda11.jar)
+* Download the [RAPIDS cuDF 21.12.2 jar](https://repo1.maven.org/maven2/ai/rapids/cudf/21.12.2/cudf-21.12.2-cuda11.jar)
```

Comment:
configs.md, getting started and others also reference the 21.12.0 cudf jar, are we going to update those as well?

Reply:
updated